vdist/profiles/centos.sh

```diff
@@ -1,14 +1,16 @@
 #!/bin/bash -x
-PYTHON_VERSION="{{compile_python_version}}"
+PYTHON_VERSION="{{python_version}}"
 PYTHON_BASEDIR="{{python_basedir}}"
 
 # fail on error
 set -e
 
 # install general prerequisites
 yum -y update
-yum install -y ruby-devel curl libyaml-devel which tar rpm-build rubygems git python-setuptools zlib-devel bzip2-devel openssl-devel ncurses-devel sqlite-devel readline-devel tk-devel gdbm-devel db4-devel libpcap-devel xz-devel
-
+yum install -y ruby-devel curl libyaml-devel which tar rpm-build rubygems git python-setuptools zlib-devel bzip2-devel openssl-devel ncurses-devel sqlite-devel readline-devel tk-devel gdbm-devel db4-devel libpcap-devel xz-devel epel-release
+yum -y install python34
```

Comment:
This seems to imply Python 3 is the only option in the centos profile. Is that intentional?